lib/main.dart

```diff
@@ -3,50 +3,61 @@ import 'dart:io';
 import 'package:destiny/views/shared/receive.dart';
 import 'package:destiny/views/shared/send.dart';
 import 'package:dart_wormhole_william/client/native_client.dart';
+import 'package:fimber/fimber.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
 import 'package:provider/provider.dart';
 import 'package:window_size/window_size.dart';
+import 'package:get_it/get_it.dart';
 
 import 'config/routes/routes_desktop_config.dart';
 import 'config/routes/routes_mobile_config.dart';
 import 'config/theme/custom_theme.dart';
 import 'constants/app_constants.dart';
+import 'locator.dart';
+
+// This is our global ServiceLocator
+GetIt getIt = GetIt.instance;
 
 void startApp(Config c) {
   WidgetsFlutterBinding.ensureInitialized();
+  Fimber.plantTree(DebugTree());
   SystemChrome.setPreferredOrientations(
       [DeviceOrientation.portraitUp, DeviceOrientation.portraitDown]);
+
+  register(getIt, c);
+
   if (Platform.isWindows || Platform.isLinux || Platform.isMacOS) {
     setWindowMinSize(const Size(900, 800));
     setWindowMaxSize(const Size(1600, 1200));
     setWindowFrame(Rect.fromLTWH(0, 0, 900, 800));
   }
   runApp(MultiProvider(providers: [
-    ChangeNotifierProvider(create: (context) => SendSharedState(c)),
-    ChangeNotifierProvider(create: (context) => ReceiveSharedState(c))
-  ], child: MyApp(c)));
+    ChangeNotifierProvider(create: (context) => SendSharedState()),
+    ChangeNotifierProvider(create: (context) => ReceiveSharedState())
+  ], child: MyApp()));
 }
 
 void main() {
+  final Config magicWormholeIO = Config(
```

Comment:
Should we really have default config magic-wormhole? 
Maybe it should be to stage.mw.leastauthority servers? Otherwise we by default might pollute common or production servers during dev. or testing.

Reply:
That is what we had before. Switching the default should be fine.